runtime/near-vm-runner/benches/bench.rs

```diff
@@ -6,10 +6,12 @@ use bencher::{benchmark_group, benchmark_main, Bencher};
 
 use near_runtime_fees::RuntimeFeesConfig;
 use near_vm_logic::mocks::mock_external::MockedExternal;
-use near_vm_logic::types::PromiseResult;
+use near_vm_logic::types::{PromiseResult, ProtocolVersion};
 use near_vm_logic::{ReturnData, VMConfig, VMContext, VMOutcome};
 use near_vm_runner::{run, VMError};
 
+const LATEST_PROTOCOL_VERSION: ProtocolVersion = ProtocolVersion::MAX;
```

Comment:
why?

Reply:
Runner doesn't have primitives, so I can't use real protocol version.

Reply:
I see. @bowenwang1996 ?

Reply:
I was wondering whether using max here can cause problems. For example if we have some function that requires doing arithmetic on protocol version it might easily overflow.